app/src/main/java/com/asma/tasky/core/domain/util/Util.kt

```diff
@@ -0,0 +1,15 @@
+package com.asma.tasky.core.domain.util
+
+import java.time.LocalDateTime
+
+object Util {
```

Comment:
I'd pick something more descriptive, because this already sounds like a violation of single-responsibility (the name `Util` implies this class will be a container for all utility functions of your app). Better something like `UserUtil`